ui/src/views/Documentation/People/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import { func } from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { contributors } from '../../../../../.all-contributorsrc';
```

Comment:
I think we can remove `json-loader` and instead do:

```jsx
import { contributors } from '../../../../../.all-contributorsrc.json';
```

Could you give it a try to see if it works, please?

Reply:
No love:
```
Module not found: Error: Can't resolve '../../../../../.all-contributorsrc.json' in '/home/dustin/p/taskcluster/ui/src/views/Documentation/People'
```

Reply:
I was able to make it work via:

```diff
diff --git a/ui/src/views/Documentation/People/index.jsx b/ui/src/views/Documentation/People/index.jsx
index ddbfd9751..335a8e831 100644
--- a/ui/src/views/Documentation/People/index.jsx
+++ b/ui/src/views/Documentation/People/index.jsx
@@ -2,7 +2,7 @@ import React, { Component } from 'react';
 import { withStyles } from '@material-ui/core/styles';
 import { func } from 'prop-types';
 import { withRouter } from 'react-router-dom';
-import { contributors } from '../../../../../.all-contributorsrc';
+import contributorsRc from '../../../../../.all-contributorsrc.json';
 
 @withRouter
 @withStyles(() => ({
@@ -43,6 +43,7 @@ export default class People extends Component {
 
   render() {
     const { classes, filter } = this.props;
+    const { contributors } = contributorsRc;
 
     return (
       <div className={classes.peopleList}>
```

Reply:
Did you also remove the config in `.neutrinorc`?  I did almost the same thing except that I kept the destructuring assignment.  Does that not work in this specific case for some reason?

Reply:
> Did you also remove the config in .neutrinorc? 

Yes

>  Does that not work in this specific case for some reason?

Seems like it doesn't. There's probably an explanation for that. I'm not sure tbh.

Reply:
I bet the explanation is #495.